pkg/network/java/testutil/JavaClientSimulator/src/main/java/Clients/JavaClients.java

```diff
@@ -0,0 +1,136 @@
+package Clients;
+
+import okhttp3.Call;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+
+import javax.net.ssl.HttpsURLConnection;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URL;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+public class JavaClients {
+
+    private static String URL_SCHEME = "https://";
+    public static void HttpsURLConnection(int iterations, String url) throws IOException, InterruptedException {
+        if (iterations == -1){
+            while (true){
+                HttpsURLConnection urlConnection =(HttpsURLConnection) new URL(URL_SCHEME+url).openConnection();
```

Comment:
Do we want to create a new connection for each request?

Reply:
fixed for httpclient and okhttp,
urlconnection is designed to use connection per request
Currently apache client is creating a connection for each request (i added todo and comment in the code)